File: Polinomio.c
#include <stdio.h>
int main(int argc, char *argv[]) {
		int x=2, t=5; 
		int x1=3, t1=-2;
		int sumax=0, sumat=0;
			printf("Programa que suma dos polinomios.\n\n");
			printf("\t(2x + 5) + (3x - 2) ");
			sumax=x+x1;
			sumat=t+t1;
			printf("= %dx + %d \n",sumax, sumat);
	return 0;
}

File: workspace/my_project/src/polinomio.scala
object polinomio {
  def main(args: Array[String]) {
  
    println ("El programa sumara dos polinomios grado 1")
    println ("De la forma (Ax+b)+(Cx-d)")
    
    println ("Introduce primer polinomio")
    println ("Introduce coeficiente de la variable 'x' ")
    var coef1: Int = readInt 
    println ("Introduce coeficiente")
    var valor1: Int = readInt 
    println ("")
    println ("Introduce segundo polinomio")
    println ("Introduce coeficiente de la variable 'x'")
    var coef2: Int = readInt 
    println ("Introduce coeficiente")
    var valor2: Int = readInt 
    var sumax:Int = coef1+coef2
    var sumay:Int = valor1+valor2
    
  println("\nLa suma de  " + coef1 + "x +" + valor1 + "\n+\n " + coef2 + "x +" + valor2)     
  println("\nEs " + sumax + "x+" + sumay + "\n\n ")  
    
 }
}